docs/quick-start.md

```diff
@@ -116,6 +123,13 @@ The pipeline can be deleted by
 kubectl delete -f https://raw.githubusercontent.com/numaproj/numaflow/stable/test/e2e/testdata/even-odd.yaml
 ```
 
+## Clean up
```

Comment:
Don't think this section needs to be mentioned particularly. BTW - regular `Ctrl+c` can terminate it.

Reply:
Agree that a regular `Ctrl+c` is better. I was using `Ctrl+z`, which didn't close the port hence made it occupied, causing "Address in use" error the very next time I played with this tutorial. I guess it should be obvious to use `Ctrl+c`. Will remove.